src/pixelator/config/config_class.py

```diff
@@ -83,11 +107,40 @@ def get_assay(self, assay_name: str) -> Optional[Assay]:
         """Get an assay by name."""
         return self.assays.get(assay_name)
 
+    def list_panel_names(self, include_aliases: bool = False) -> List[str]:
+        """Return a list of all panel names.
+
+        :param include_aliases: Include panel aliases in the list
+        :returns: A list of panel names
+        """
+        out = sorted(list(self.panels.keys()))
+
+        if not include_aliases:
+            return out
+
+        out += sorted(list(self.panel_aliases.keys()))
```

Comment:
I thought perhaps it could be nice if names and aliases are listed together. Something like this?

```
pixelator single-cell --list-panels
human-sc-immunology-spatial-proteomics-1 (aliases: human-sc-immunology-spatial-proteomics, apa, monkey)
human-sc-immunology-spatial-proteomics-2
```

Not the cleanest implementation, but here is a suggestion:

```suggestion
    def list_panel_names(self, include_aliases: bool = False) -> List[str]:
        """Return a list of all panel names.

        :param include_aliases: Include panel aliases in the list
        :returns: A list of panel names
        """

        plain_out = sorted(list(self.panels.keys()))

        if not include_aliases:
            return plain_out

        def get_names_to_aliases():
            names_to_aliases = defaultdict(list)
            for aliases, name in self.panel_aliases.items():
                names_to_aliases[name].append(aliases)
            return names_to_aliases

        panel_names_to_aliases = get_names_to_aliases()

        out_with_aliases = []
        for name in plain_out:
            aliases = panel_names_to_aliases.get(name) or []
            if not aliases:
                out_with_aliases.append(name)
                continue
            aliases_str = ", ".join(aliases) if aliases else ""
            out_with_aliases.append(f"{name} (aliases: {aliases_str})")

        return out_with_aliases
```

Reply:
100% agree, but this is currently mainly used in nf-core/pixelator to validate the possible options passed in the `panel` field. 

I was thinking of making the "nice output" version the default and adding something like `--raw` and use that in nf-core/pixelator but wanted to do that in a separate PR

Reply:
Aha! I see. Sounds reasonable.